backend/minors/ART.yaml

```diff
@@ -0,0 +1,127 @@
+---
+type: Minor
+name: History of Art
+code: ART
+degree:
+description: The Department of Art and Archaeology studies artwork and 
+  material artifacts from a wide range of cultures and periods, approaching 
+  them in historical perspective. Art history’s subjects include drawing, 
+  painting, sculpture, photography, architecture and the built environment, 
+  ceramics, lacquer, metalwork, glass and textiles; along with a wide range 
+  of images and phenomena or experiences shaped by artistic craft, including
+  cinema, performance, print culture, urbanism, scientific illustration and 
+  mass media. Students of art history learn to translate observations into 
+  linguistic expression, develop visual and experiential memory and make 
+  connections among a wide array of historical evidence. They practice 
+  techniques for analyzing and interpreting the visual and material world 
+  by actively engaging primary sources—often through the direct study of 
+  original works in Princeton’s collections and local museums.
+
```

Comment:
Add "< br > < br >" for new lines. For reference, you can check out this [PHY.yaml](https://github.com/HoagieClub/plan/pull/13/files)